doc/developer/webservices.rst

```diff
@@ -12,6 +12,9 @@ Parameters
 ----------
 
 * ``version``: ``1`` or ``2 ``, API version, default is ``1``.
+* ``sets``: with kind of data we want to get, can be ``all``, ``themes``, ``group``
+  or ``background_layers``, default is ``all``.
+* ``background_layers_group``: get only one group, used to get the base layers.
 * ``group``: get only one group, used to get the base layers.
```

Comment:
??
For baselayers, shouldn't the user use the ``background_layers_group`` param?